pkgs/applications/video/obs-studio/wrapper.nix

```diff
@@ -6,19 +6,24 @@ symlinkJoin {
   name = "wrapped-${obs-studio.name}";
 
   nativeBuildInputs = [ makeWrapper ];
-  paths = [ obs-studio ] ++ plugins;
+  paths = [ obs-studio ];
 
   postBuild = with lib;
     let
       # Some plugins needs extra environment, see obs-gstreamer for an example.
       pluginArguments =
         lists.concatMap (plugin: plugin.obsWrapperArguments or []) plugins;
 
+      pluginsJoined = symlinkJoin {
+        name = "obs-plugins";
+        paths = plugins;
+      };
+
       wrapCommand = [
           "wrapProgram"
           "$out/bin/obs"
-          ''--set OBS_PLUGINS_PATH "$out/lib/obs-plugins"''
-          ''--set OBS_PLUGINS_DATA_PATH "$out/share/obs/obs-plugins"''
+          ''--set OBS_PLUGINS_PATH "${pluginsJoined}/lib/obs-plugins"''
+          ''--set OBS_PLUGINS_DATA_PATH "${pluginsJoined}/share/obs/obs-plugins"''
```

Comment:
:thinking: So, I caught something which I don't consider a "blocking" issue. But, we may hit some conflicts here since the derivation, produced in `pluginsJoined`, have more than the needed directories. For example, with `obs-vkcatpure` it ends adding the `bin/obs-*capture` files to the derivation.

What do you think about having `pluginsJoined` and `pluginsDataJoined`, mapping the list to append `/lib/obs-plugins` or `/share/obs/obs-plugins` to the paths?

Reply:
That makes sense to me. I'll give it a spin in the morning and shout back. Let me know if you see anything else!

Reply:
I lied, couldn't leave it alone for the night. Rebased in and tested a quick pass. Duplicate loads still gone, loading my plugins still works. Let me know what you think.

Reply:
Conventions Q: Are fixed names reasonable for the new plugins derivations ("obs-studio-plugins", etc), or would I be better off appending to obs-studio.name?